src/pages/api/v1/applications/index.ts

```diff
@@ -0,0 +1,46 @@
+import prisma from '../../../../lib/prisma';
+import { NextApiRequest, NextApiResponse } from 'next';
+
+const getApplications = async (req: NextApiRequest, res: NextApiResponse) => {
+  try {
+    const data = await prisma.application.findMany();
+    if (data) {
+      res.status(200).json(data);
+    } else {
+      res
+        .status(404)
+        .json({ message: `Could not find Applications try again` });
+    }
```

Comment:
Can you please reorder this so that the error response goes first and we early exit?